packages/pros/examples/adi.rs

```diff
@@ -0,0 +1,22 @@
+#![no_std]
+#![no_main]
+
+use pros::prelude::*;
+use pros::adi::{
+    AdiSlot,
+    motor::AdiMotor,
+    ultrasonic::AdiUltrasonic
+};
+
+#[derive(Default)]
+pub struct Robot;
+#[async_trait]
+impl AsyncRobot for Robot {
+    async unsafe fn opcontrol(&mut self) -> pros::Result {
+        let mut pot = AdiPotentiometer::new(1);
```

Comment:
Shouldn't this pass in `AdiSlot::A` and not `1`? This shouldn't compile, unless I'm missing something.